src/lexer.in.cc

```diff
@@ -133,10 +130,10 @@ Lexer::Token Lexer::ReadToken() {
     simple_varname = [a-zA-Z0-9_-]+;
     varname = [a-zA-Z0-9_.-]+;
 
-    [ ]*"#"[^\000\n]*"\n" { continue; }
-    [ ]*"\r\n" { token = NEWLINE;  break; }
-    [ ]*"\n"   { token = NEWLINE;  break; }
-    [ ]+       { token = INDENT;   break; }
+    [ \t]*"#"[^\000\n]*"\n" { continue; }
+    [ \t]*"\r\n" { token = NEWLINE;  break; }
+    [ \t]*"\n"   { token = NEWLINE;  break; }
+    [ \t]+       { token = INDENT;   break; }
```

Comment:
Thanks. I need this is not enough since you also need to handle line continuations.

It looks like this requires also modifying the definitions of EatWhiteSpace and ReadEvalString below, for example modify `$\r\n"[ ]*` into `$\r\n"[ \t]*` and `$\n"[ ]*` into `$\r\n"[ ]*`.